### PR TITLE
Add server, route, controller, model, and database logic

### PR DESCRIPTION
**Pull request by rroque98**

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -1,45 +1,50 @@
...
-      accuracy_rating: Number,
-      communication_rating: Number,
-      cleanliness_rating: Number,
-      location_rating: Number,
-      check_in_rating: Number,
-      value_rating: Number,
-      is_reported: Boolean,
-  });
 
+let reviewSchema = new mongoose.Schema({
```
Comment (marcellino-ornelas) [inline db/index.js]: schema should never change so we can use `const` right here

File: db/index.js [hunk 2]
```diff
@@ -1,45 +1,50 @@
...
+  location_rating: Number,
+  check_in_rating: Number,
+  value_rating: Number,
+  is_reported: Boolean,
+});
+
+const Review = mongoose.model('Review', reviewSchema);
 
-  const Review = mongoose.model('Review', reviewSchema);
+const save = (data) => {
```
Comment (marcellino-ornelas) [inline db/index.js]: Are you still using this?

**Review by marcellino-ornelas — DISMISSED** [review]
Go back and clean code a little more. There is a lot of code that we moved into other files that still exist in file.
i would also delete db folder. it is no longer getting used. move code in dataGenerator to generate.js

**Comment by rroque98** [comment]
You are right. I did not need that folder any longer after closer inspection. I have deleted it now. I left relevant data generator code in generate.js but commented it out since I have loaded my database with the data I need for now. I will come back to work on generate.js at a future point.

**Review by marcellino-ornelas — APPROVED** [review]

**Pull request merged**